skrf/network.py

```diff
@@ -2717,6 +2718,14 @@ def extrapolate_to_dc(self, points: int = None, dc_sparam: NumberLike = None, ki
         impulse_response
         step_response
         """
+        if kind == 'warn':
+            warnings.warn("The default `kind` parameter will change "
+                          "from `rational` to `cubic` in future versions."
+                          "Use `kind='rational'` to keep the old default."
+                          "To silent this warning, explitly define `kind`.",
+                          category=DeprecationWarning, stacklevel=2)
```

Comment:
Did you figure out what `stacklevel=2` does? I know, [the Python doc](https://docs.python.org/3/library/warnings.html#warnings.warn) says *"The stacklevel argument can be used by wrapper functions written in Python"*, but it that important?

Reply:
I found out that `stacklevel=1` (default) didn't display any message in ipython... so I've used 2.